cluster-autoscaler/cloudprovider/aws/examples/cluster-autoscaler-autodiscover.yaml

```diff
@@ -157,7 +157,7 @@ spec:
             - --node-group-auto-discovery=asg:tag=k8s.io/cluster-autoscaler/enabled,k8s.io/cluster-autoscaler/<YOUR CLUSTER NAME>
           volumeMounts:
             - name: ssl-certs
-              mountPath: /etc/ssl/certs/ca-certificates.crt
+              mountPath: /etc/ssl/certs/ca-certificates.crt #/etc/ssl/certs/ca-bundle.crt for EKS Amazon Linux Worker Nodes
```

Comment:
I believe this will be the case for any AmazonLinux worker nodes, so you can probably just remove the EKS part.

Reply:
@mmerkes I updated as your suggestion already.

Reply:
Shouldn't this comment be moved to `volumes: hostPath:` part? IIUC, the difference comes from the host OS, not from the container.